feat(DOC-1970): add missing rpk connect reference pages (#1608)
redpanda-data/documentation: modules/reference/pages/rpk/rpk-connect/rpk-connect-agent-run.adoc

= rpk connect agent run
:description: Execute a Redpanda Connect agent as part of a pipeline with MCP tool access.

[IMPORTANT]
====
This command is experimental and subject to change.
====

> Execute a Redpanda Connect agent as part of a pipeline that has access to tools via the MCP (Model Context Protocol). Each resource in the `mcp` subdirectory creates tools that can be used, then the `redpanda_agents.yaml` file along with Python agent modules are invoked.

Reviewer: ```suggestion
Execute a Redpanda Connect agent as part of a pipeline that has access to tools via the MCP (Model Context Protocol). The command first reads resource definitions in the `mcp` subdirectory and exposes them as tools. It then runs the agent defined in `redpanda_agents.yaml` along with any Python agent modules.
```
The second sentence is a bit unclear to me, so here's a suggestion to rephrase -- @mmatczuk please let us know if the rephrase is correct.

== Usage

 rpk connect agent run [OPTIONS]

== Example

[,bash]
----
rpk connect agent run ./repo
----

== Options

[cols="1m,2a"]
|===
| Option | Description

> | --secrets env:

Reviewer: Based on the description, if `env:` is the default, but you could use `--secrets none:`, maybe we should keep the string in this cell to `--secrets` only? I'd also suggest adding explicit usage examples in the preceding section, it's not very clear to me how to use this option

| Attempt to load secrets from a provided URN. If more than one entry is specified, they are attempted in order until a value is found. Environment variable lookups are specified with the URN `env:`, which by default is the only entry. To disable all secret lookups, specify a single entry of `none:` (default: `env:`).

| --redpanda-license
| Provide an explicit Redpanda License, which enables enterprise functionality. By default, licenses found at the path `/etc/redpanda/redpanda.license` are applied.

| --help, -h
| Show help for the command.

> 4+h| Linux only

Reviewer: Please double check what this is for, it's currently not being rendered: https://deploy-preview-1608--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-connect/rpk-connect-agent-run/#options If it's meant to indicate that the following options are for Linux only, we should just indicate that in the individual description cells and remove this row altogether.

| --chroot

> | Chroot into the provided directory after parsing configuration. Common `/etc/` files are copied to the chroot directory, and the directory is made read-only.

Reviewer: ```suggestion
| (Linux only) Chroot into the provided directory after parsing configuration. Common `/etc/` files are copied to the chroot directory, and the directory is made read-only.
```
Based on previous [suggestion](https://github.com/redpanda-data/docs/pull/1608/changes#r2967824477)

| --chroot-passthrough

> | Specify additional files to be copied into the chroot directory. Can be specified multiple times. Only valid when `--chroot` is used.

Reviewer: ```suggestion
| (Linux only) Specify additional files to be copied into the chroot directory. Can be specified multiple times. Only valid when `--chroot` is used.
```
Based on previous [suggestion](https://github.com/redpanda-data/docs/pull/1608/changes#r2967824477)
